src/lib/util/cdrom.cpp

```diff
@@ -657,21 +657,11 @@ void cdrom_file::get_info_from_type_string(const char *typestring, uint32_t *trk
 		*trktype = CD_TRACK_MODE2_FORM_MIX;
 		*datasize = 2336;
 	}
-	else if (!strcmp(typestring, "MODE2/2336"))
-	{
-		*trktype = CD_TRACK_MODE2_FORM_MIX;
-		*datasize = 2336;
-	}
 	else if (!strcmp(typestring, "MODE2_RAW"))
 	{
 		*trktype = CD_TRACK_MODE2_RAW;
 		*datasize = 2352;
 	}
-	else if (!strcmp(typestring, "MODE2/2352"))
-	{
-		*trktype = CD_TRACK_MODE2_RAW;
-		*datasize = 2352;
-	}
```

Comment:
There’s no other place that catches `MODE2/2352` so it isn’t a duplicate.  Why did you remove it?

Reply:
oh sorry i confused it with MODE1/2352